README.md

```diff
@@ -209,16 +176,26 @@ and improve:
 ![Architecture](docs/img/modin_architecture.png)
 
 Visit the [Documentation](https://modin.readthedocs.io/en/latest/development/architecture.html) for
-more information, and checkout [the difference between Modin and Dask!](https://github.com/modin-project/modin/tree/master/docs/modin_vs_dask.md)
+more information!
 
 **`modin.pandas` is currently under active development. Requests and contributions are welcome!**
 
 
-### More information and Getting Involved
+### General Resources and Getting Involved
+
+- [Documentation Link](https://modin.readthedocs.io/en/latest/)
+- [Modin Examples + Tutorials](https://modin.readthedocs.io/en/latest/getting_started/examples.html)
+- [Modin Talks + Blogs](https://modin.readthedocs.io/en/latest/getting_started/examples.html#talks-podcasts)
+- [Modin Slack](https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA)
+- [Modin Discourse](https://discuss.modin.org)
+- [Modin GitHub Issues Page](https://github.com/modin-project/modin/issues)
+- [Modin FAQs](https://modin.readthedocs.io/en/latest/getting_started/faq.html)
+- [Modin Troubleshooting Guide](https://modin.readthedocs.io/en/latest/getting_started/troubleshooting.html)
+- [Modin Paper](https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf)
+
+For more information on how to contribute to Modin, check out the 
```

Comment:
Why is the contribution guide separate from everything else?

Reply:
Moved the modin.pandas is currently under active development bold and italic part down here to connect the two.